clang/test/Sema/warn-compare-enum-types-mismatch.c

```diff
@@ -1,12 +1,19 @@
 // RUN: %clang_cc1 -x c -fsyntax-only -verify -Wenum-compare -Wno-unused-comparison %s
 // RUN: %clang_cc1 -x c++ -fsyntax-only -verify -Wenum-compare -Wno-unused-comparison %s
 
+// In C enumerators (i.e enumeration constants) have type int (until C23). In order to support diagnostics such as 
+// -Wenum-compare we pretend they have the type of their enumeration.
+
 typedef enum EnumA {
   A
 } EnumA;
 
 enum EnumB {
-  B
+  B,
+  B1 = 1,
+  // In C++ this comparison doesnt warn as enumerators dont have the type of their enumeration before the closing 
```

Comment:
Same issue here as above with trailing whitespace and 80 col.

Reply:
done